modules/configuring-a-custom-notebook.adoc

```diff
@@ -26,15 +26,16 @@ endif::[]
 .Procedure
 . From the {productname-short} dashboard, click *Settings* -> *Notebook images*.
 +
-The *Notebook image settings* page appears. Previously imported notebook images are displayed. To enable or disable a previously imported notebook image, on the row containing the relevant notebook image, click the toggle in the *Enable* column.
+The *Notebook image settings* page appears. Previously imported notebook images are displayed. To enable or disable a previously imported notebook image, on the row containing the relevant notebook image, click the toggle in the *Enable* column. Click *Create profile* on the row containing the relevant notebook image to create an accelerator profile as the image's recommended accelerator.  
```

Comment:
If the imported image has no Accelerator identifier, that button should be disabled. When hovering ,it shows the text "To create an accelerator profile for this image, edit it to include an accelerator identifier."

Reply:
Fixed.